language/move-lang/src/expansion/translate.rs

```diff
@@ -782,6 +791,22 @@ fn exp_(context: &mut Context, sp!(loc, pe_): P::Exp) -> E::Exp {
             }
             None => EE::Name(n),
         },
+        PE::GenericName(pn, ptys_opt) => {
+            if !context.require_spec_context(loc, "`n<..>` or `m::n<..>` variable expression") {
```

Comment:
I'm not thrilled with this "n\<..\>" stuff in the error message. It's also not clear that this is supposed to be a variable expression. We use the "name\<..\>" syntax for functions as well as variables. Could we do something custom here and report the same error message as before (i.e., without referring to specifications)?

Reply:
Great catch @bob-wilson

We should probably match this to the other unexpected name error messages.

Reply:
Done (same error message like before now, but with out parsing errors)